Vagrantfile

```diff
@@ -7,6 +7,7 @@ VAGRANTFILE_API_VERSION = '2'
 Vagrant.configure(VAGRANTFILE_API_VERSION) do |config|
   config.vm.box = 'ubuntu/xenial64'
   config.vm.hostname = "#{HOSTNAME}"
+  config.vm.network "forwarded_port", guest: 22, host: 2200, auto_correct: true
```

Comment:
I was sure Vagrant tries to fix any SSH port collision by default.

If that wouldn't happen, then we'd change all Vagranfiles we have in all repos:
https://github.com/search?q=org%3AStackStorm+Vagrantfile&type=Code including https://github.com/StackStorm/st2vagrant/blob/master/Vagrantfile

So for me it seems like we have some unexpected Vagrant usage here.
How do you run this Vagrant VM?
What's your Vagrant version?

This also might help to understand the problem better:
http://stackoverflow.com/a/24131225/4533625
http://stackoverflow.com/a/25142933/4533625